src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {

```

Comment:
Don't use static methods in your solution. Instead of making `getRandomBall` a static method, you should create an instance of the Lottery class where you need to use this method.